contrib/debian/rules

```diff
@@ -12,10 +12,16 @@ override_dh_auto_clean:
 	if [ -f Makefile ]; then $(MAKE) distclean; fi
 	rm -rf Makefile.in aclocal.m4 configure src/Makefile.in src/bitcoin-config.h.in src/build-aux src/qt/Makefile.in src/qt/test/Makefile.in src/test/Makefile.in
 
+ifeq ($(shell dpkg-vendor --derives-from Ubuntu && echo yes),yes)
```

Comment:
Wait, the dpkg-vendor version doesn't work, or the if doesnt hold the variable outside of the scope?